### PR TITLE
update getting started doc

### LINKED ISSUE #91: Add an introduction to Getting started
The current [index page for getting started](https://www.opencue.io/docs/getting-started/) is a useful URL to link to from other sites that want to show users how to install OpenCue.

This page should provide more context to explain to new users that it might be useful to first read the overview and become familiar with common OpenCue terminology. We should explain the following:

- What the guides in this section cover
- There are a lot of components in OpenCue
- It might be useful to understand how all the components work together (link to [overview](https://www.opencue.io/docs/concepts/opencue-overview/))
- It might be useful to refer to the glossary as you work through the guides (link to [glossary](https://www.opencue.io/docs/concepts/glossary/))

Note: Most of the contents of this page are auto-generated. Anything added to the Markdown file will appear before the auto-generated index of guides.

### PR DESCRIPTION
**Pull request by apoorvkhare07**
Fix #91 by adding an introduction and other relevant links to getting started doc.

### CODE REVIEW

**Comment by sharifsalah** [comment]
Thanks for sending the pull request @apoorvkhare07. Unfortunately, this isn't quite right as you've included a lot of changes outside of the scope of the original issue. There should only be a single Markdown file in this pull request.

To give you some general feedback about your suggestion, this project is built on the open source Docsy project: https://github.com/google/docsy-example

You can see that Docsy organizes documentation by type, such as conceptual, how-to, getting started. The getting started documentation should be kept to a minimum: 

https://example.docsy.dev/docs/

If you are interested and have experience, there is lots of work to do for getting started related to Windows installation. However I don't think it is a good idea to mix concepts and getting started.

I hope that helps.

**Comment by apoorvkhare07** [comment]
ok @sharifsalah. Looks like I misunderstood the issue. 
So what you are asking is to only change the getting started's index.md file to include the new detailed introduction and the new links you mentioned in the issue ?

**Comment by sharifsalah** [comment]
Yes @apoorvkhare07, that sounds like a good description of the issue. Thanks for confirming your understanding.

**Comment by apoorvkhare07** [comment]
@sharifsalah please review now, I have updated the PR accordingly.

**Comment by sharifsalah** [comment]
I had a quick look @apoorvkhare07 and I can still see 5 files included in this pull request. Please update the pull request so that only content/docs/Getting started/_index.md is included.

**Comment by apoorvkhare07** [comment]
@sharifsalah Updated the PR.

File: content/docs/Getting started/_index.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ date: 2019-02-22
 description: >
   Guides for system admins deploying OpenCue components and installing dependencies
 ---
+
+There are a lot of components in OpenCue. If you are new to OpenCue it might be useful to understand how all the components work together in the [overview](https://www.opencue.io/docs/concepts/opencue-overview/) section.
```
Comment (sharifsalah) [inline content/docs/Getting started/_index.md]: I recommend updating to:

The guides in this section provide detailed instructions for installing OpenCue in a production environment. If you'd like to quickly try out OpenCue to evaluate the project for your needs, see the [Quick starts](/docs/quick-starts/).

OpenCue relies on a lot of components and project-specific terminology. If you're new to OpenCue, you might find it useful to learn how some of the components work together. For more information, see the [OpenCue overview](/docs/concepts/opencue-overview/) and [Glossary](/docs/concepts/glossary/).

Comment (apoorvkhare07) [reply content/docs/Getting started/_index.md]: updated. please check.

Comment (sharifsalah) [reply content/docs/Getting started/_index.md]: Looks good @apoorvkhare07, thank you.

**Pull request merged**